lib/perl/Genome/Config.pm

```diff
@@ -130,6 +130,12 @@ sub all_keys {
 
 sub config_subpath { Path::Class::File->new('genome', 'config.yaml') }
 
+sub project_dir {
+    my $dir = $ENV{XGENOME_CONFIG_PROJECT_DIR};
+    return $dir if $dir;
```

Comment:
Mine matches `snapshot_dir` (which also differs from `global_dirs`).  You're right, though--it's odd that they're inconsistent!  That should be reconciled one way or the other...